tests/integration/targets/azure_rm_postgresqlconfiguration/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+- name: Prepare random number
+  set_fact:
+    rpfx: "{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
```

Comment:
basically same comments as mysql:
1. combine xxxconfiguration test as alias of main module
2. update teste
3.  var definition